index.html

```diff
@@ -5101,21 +5112,23 @@ <h3>User Agent Processing</h3>
 													href="https://infra.spec.whatwg.org/#nulls">null</a>.</li>
 											<li>If the element has a <code>rel</code> attribute, and the value of the
 												attribute is not an empty string after trimming leading and trailing
-												white space, set <var>current_toc_branch["rel"]</var> to the trimmed
-												value. Otherwise, set <var>current_toc_branch["rel"]</var> to <a
+												white space, split the trimmed value on whitespace and set
+													<var>current_toc_branch["rel"]</var> to the resulting <a
+													href="https://infra.spec.whatwg.org/#list">list</a> of tokens.
+												Otherwise, set <var>current_toc_branch["rel"]</var> to <a
```

Comment:
Github does not let me comment on unchanged lines; but it is a consequence of this change: In step (4), where `current_toc_branch` is initialized, I would expect the `rel` value to be set to either an empty array or, most probably, to `null`.

Reply:
It looks like this has already been fixed, as I see rel -> null in step 4.

I'm fine with that, but then `type` should probably be set to null as well for consistency. And in both cases, the "otherwise" statement at the end becomes redundant so can be removed (both values will already be null if their respective conditions to add a value aren't met).

Reply:
I've actually switched all the properties to null by default.

The only one that affects processing is `name`, but it took me a while to figure out what it was doing. Instead of starting it off as empty string and then setting it to null if a name is an empty string, we should start it off as null and just leave an empty string if that's the result.

Setting it to null conflicts with 5.2, which is where a placeholder gets inserted when the name is an empty string.